.changeset/eighty-owls-cheat.md

```diff
@@ -0,0 +1,26 @@
+---
+"@comet/admin-theme": major
+---
+
+Rework colors
+
+- Rename `bluePalette` to `primaryPalette`
+- Rename `neutrals` to `greyPalette`
+- Remove `greenPalette`
+- Remove `secondary` from `paletteOptions`
+- Change colors in all palettes
+- Change `text` colors 
+- Add `highlight` colors `purple`, `green`, `orange`, `yellow` and `red` to palette
+
+Hint: To use the `highlight` colors without getting a type error, you must adjust the `tsconfig.json` in your project:
```

Comment:
Should we add a code-mod for this?

Reply:
We should first find a better solution for this: https://github.com/vivid-planet/comet/pull/1840#discussion_r1526500119.

Reply:
Done, see https://github.com/vivid-planet/comet/pull/1840#discussion_r1539711357